### DISCUSSION TITLE
Unable to Login - 403 Forbidden

### DISCUSSION THREAD

**Discussion by noespa** [orchidsoftware/platform#2144] [General]
Hello,

I have app write at the beginning in Laravel 6, next in 2020 I update Laravel to 8 (I wanted to run Livewire 2) and now update Laravel to 9. Now I install Orchid and I can't log in to admin panel. I see Orchid login page and create admin user with command:
`php artisan orchid:admin admin admin@admin.com password`
After login on Orchid login page i see only 403 Forbidden. When I go to main page of app i am log in to this account.
I try change in platform.php config this line:
`'auth'  => true/false,`
and still the same. After login i see 403. In Laravel log i can't see any problem. 

I comment out all route in web.php and clear app cache. The same problem - 403 Forbidden.

Can u help me?

Sorry for my English :)

**Comment by ppalmeida** [comment]


**Reply by tabuna** [reply]
Hello @ppalmeida. It seems to me that you are mistaken, since it is assumed that you will write the password not in a hashed form. Here is the method responsible for this

https://github.com/orchidsoftware/platform/blob/2d171616d870689a401a33d88d04f56518cd0cc6/src/Platform/Models/User.php#L97-L107

It is possible that you have an observer or mutator that does the processing automatically, but this is definitely not the default. And if so, then you can override this method in your model.

**Reply by ppalmeida** [reply]
Oh, really sorry. So, this is deff something with my app. I'll remove/edit my comment, so it won't cause any confusion.
